test/test_collection.py

```diff
@@ -2178,6 +2178,24 @@ def test_bool(self):
         with self.assertRaises(NotImplementedError):
             bool(Collection(self.db, 'test'))
 
+    @client_context.require_version_min(5, 0, 0)
+    def test_helpers_with_let(self):
+        c = self.db.test
+        helpers = [("delete_many", ({}, {})), ("delete_one", ({}, {})),
```

Comment:
Please refactor this to use the methods instead of strings, eg:
```
helpers = [(c.delete_many,...
```

It makes it easier to find usages of the method.

Reply:
Done.